docs/Getting-Started/polykey-cli/installation.md

```diff
@@ -83,16 +93,18 @@ a terminal to restart it.
 - For ZSH:
 
 ```shell
-export PATH=~/Downloads:$PATH >> ~/.zshrc && source ~/.zshrc
+export PATH="$HOME/Downloads:$PATH"' >> ~/.zshrc && source ~/.zshrc
 ```
 
 - For Bash:
 
+– For Bash:
+
 ```shell
-export PATH=~/Downloads:$PATH >> ~/.bashrc && source ~/.bashrc && source
+echo 'export PATH="$HOME/Downloads:$PATH"' >> ~/.bashrc && source ~/.bashrc
```

Comment:
The single quote here is correct due to the echo.

Beware about this, make sure to write these docs critically.

Also I wish you to try these commands when you write these docs, don't write docs blindly, ALWAYS SANITY CHECK.